janus_core/calculations/phonons.py

```diff
@@ -237,8 +237,8 @@ def __init__(
             End temperature for thermal calculations, in K. Default is 1000.0.
         temp_step
             Temperature step for thermal calculations, in K. Default is 50.0.
-        force_consts_to_hdf5
-            Whether to write force constants in hdf format or not. Default is True.
+        hdf5
```

Comment:
Is it definitely fine having these as a combined option, rather than two separate options?